### ISSUE TITLE
cloud_monitoring list_alarms - TypeError: __init__() takes at least 4 arguments (4 given)

### ISSUE THREAD

**Issue by ghost** [rackspace/pyrax#566]
```
cm = pyrax.cloud_monitoring
lst_entities = cm.list_entities()
for entity in lst_entities:
    print entity.id
    lst_alarms = cm.list_alarms(entity.id)    #this call seem to have some issue
    print lst_alarms
    break
```
Getting below error, not sure if I am doing something wrong or there is a bug. Please help.
```
Traceback (most recent call last):
  File "./pyrax_cm_alarms.py", line 31, in <module>
    lst_alarms = cm.list_alarms(entity.id)
  File "/opt/build/lib/python2.7/site-packages/pyrax/cloudmonitoring.py", line 63, in _wrapped
    return fnc(self, entity, *args, **kwargs)
  File "/opt/build/lib/python2.7/site-packages/pyrax/cloudmonitoring.py", line 1335, in list_alarms
    return_next=return_next)
  File "/opt/build/lib/python2.7/site-packages/pyrax/cloudmonitoring.py", line 237, in list_alarms
    return_next=return_next)
  File "/opt/build/lib/python2.7/site-packages/pyrax/cloudmonitoring.py", line 275, in list
    marker=marker, **kwargs)
  File "/opt/build/lib/python2.7/site-packages/pyrax/manager.py", line 89, in list
    return self._list(uri, return_raw=return_raw, other_keys=other_keys)
  File "/opt/build/lib/python2.7/site-packages/pyrax/manager.py", line 159, in _list
    ret = [obj_class(self, res, loaded=False) for res in data if res]
TypeError: __init__() takes at least 4 arguments (4 given)
```

**Comment by p-pawlak** [comment]
same change introduced in 314ab36

**Comment by jamiehannaford** [comment]
Fixed by #617
